### PR TITLE
Feat/add russian super glue support

### PR DESCRIPTION
**Pull request by vaskonov**
Feat/add russian super glue support

### CODE REVIEW

**Review by IgnatovFedor — CHANGES_REQUESTED** [review]

File: docs/features/models/superglue.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+Russian SuperGlue Submission
+==========================================
+The DeepPavlov library provides a way to train your Russian SuperGlue model and submit the results to the leaderboard in a couple of easy steps.     
+
+Task definition
+---------------
+`Russian SuperGlue <https://russiansuperglue.com/>`__ is a benchmark that contains a set of tasks in Russian developed for evaluating a general language understanding.
```
Comment (mtalimanchuk) [inline docs/features/models/superglue.rst:7]: for evaluating general language understanding

File: docs/features/models/superglue.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+There are 9 tasks in the Russian SuperGlue set:
+
+`DaNetQA (Yes/no Question Answering Dataset for the Russian)` is a binary classification task of question answering, in which the model is asked to answer a yes/no question based on a given context fragment.
```
Comment (mtalimanchuk) [inline docs/features/models/superglue.rst:11]: for Russian

File: docs/features/models/superglue.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+In the `MuSeRC (Russian Multi-Sentence Reading Comprehension)` task the model needs to process information from multiple sentences at once and identify the correct answers for the 
+question from the given list.
+
+In the `RuCoS (Russian reading comprehension with Commonsense reasoning)` task the model has to choose the answer to each query from a list of text spans from a fragment.
+
+`RUSSE (Russian Word-in-Context)` is a reading comprehension task in which the model has to identify whether a given word is used in the same 
+meaning in two different sentences.
+
+In `RWSD (The Russian Winograd Schema Challenge)` the data is a set of sentences that differ with one or two words
```
Comment (mtalimanchuk) [inline docs/features/models/superglue.rst:25]: is a set of sentences that differ by one or two words

File: tests/test_quick_start.py [hunk 4]
```diff
@@ -122,6 +122,13 @@
         ("classifiers/superglue/superglue_boolq_roberta_mnli.json", "classifiers", ('TI',)): [TWO_ARGUMENTS_INFER_CHECK],
         ("classifiers/superglue/superglue_record_roberta.json", "classifiers", ('TI',)): [RECORD_ARGUMENTS_INFER_CHECK]
     },
+    "russian_super_glue":{
```
[the remark this answers is not in the document]
Comment (vaskonov) [reply tests/test_quick_start.py:125]: parus test has been added

File: deeppavlov/dataset_readers/huggingface_dataset_reader.py [hunk 5]
```diff
@@ -355,3 +455,34 @@ def add_num_examples(dataset: Dataset) -> Dict[str, List[int]]:
...
+
+    contexts: List[str] = []
+
+    for paragraph, question in zip(paragraphs, questions):
+        contexts.append(f"{paragraph} {question}")
+
+    return {"context": contexts}
+
+
+def preprocess_wsc(dataset: Dataset) -> Dict[str, List[str]]:
```
[the remark this answers is not in the document]
Comment (vaskonov) [reply deeppavlov/dataset_readers/huggingface_dataset_reader.py:482]: added

File: deeppavlov/dataset_readers/huggingface_dataset_reader.py [hunk 6]
```diff
@@ -355,3 +455,34 @@ def add_num_examples(dataset: Dataset) -> Dict[str, List[int]]:
     """
     num_examples = len(dataset[next(iter(dataset))])
     return {"num_examples": [num_examples] * num_examples}
+
+
+def preprocess_multirc(examples: Dataset, *, clean_paragraphs: bool = True) -> Dict[str, List[str]]:
```
[the remark this answers is not in the document]
Comment (vaskonov) [reply deeppavlov/dataset_readers/huggingface_dataset_reader.py:460]: done

**Pull request closed without merging**